demo-shell/src/app/components/cloud/process-list-cloud-example.component.html

```diff
@@ -1,80 +1,59 @@
 <div>PROCESS LIST CLOUD</div>
```

Comment:
can you use the i18n in this file ?

Reply:
this is just a temporary page the official one will be created with https://issues.alfresco.com/jira/browse/ADF-3775